docs/client-options.md

```diff
@@ -56,6 +56,7 @@ third party clients.
 
 ## Java
 
+- [Key Manager Plus](https://www.manageengine.com/key-manager/) (End-to-end Certificate Life-cycle Management)
```

Comment:
Two pieces of feedback:
1. Could you move this line beneath the existing `porunov/acme_client` entry.
2. Can you also remove "(End-to-end Certificate Life-cycle Management)"? By definition an ACME client is designed to provide end-to-end certificate life cycle management :-)